src/overlays/actors/ovl_Bg_Mizu_Bwall/z_bg_mizu_bwall.c

```diff
@@ -21,23 +28,435 @@ const ActorInit Bg_Mizu_Bwall_InitVars = {
     (ActorFunc)BgMizuBwall_Update,
     (ActorFunc)BgMizuBwall_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Mizu_Bwall/func_8089C480.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Mizu_Bwall/BgMizuBwall_Init.s")
+static ColliderTrisItemInit sTrisElementInitFloor[2] = {
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { { { -40.0f, 0.0f, -40.0f }, { -40.0f, 0.0f, 40.0f }, { 40.0f, 0.0f, 40.0f } } },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { { { -40.0f, 0.0f, -40.0f }, { 40.0f, 0.0f, 40.0f }, { 40.0f, 0.0f, -40.0f } } },
+    },
+};
+
+static ColliderTrisInit sTrisInitFloor = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x00, 0x00, COLSHAPE_TRIS },
+    ARRAY_COUNT(sTrisElementInitFloor),
```

Comment:
afaik we dont use array count for this in most places, seems like something we should do all at once